x-pack/plugins/endpoint/common/types.ts

```diff
@@ -155,6 +159,20 @@ export interface EndpointEvent {
   agent: {
     type: string;
   };
+  endgame: {
+    pid: number;
```

Comment:
❔ Since we're bringing these things in, it might brook a short explanation in comments (Like why is a pid not the same as a unique_pid? - things we know, but maybe are not obvious to someone who hasn't worked with us before that might be expected to read and understand this.)